DEVSETUP.md

```diff
@@ -142,3 +142,15 @@ SCHEMAS_CONFIGMAP_NAME=identity-schemas \
 SCHEMAS_CONFIGMAP_NAMESPACE=default \
 TRACING_ENABLED=false \
 ```
+
+
+## OIDC authentication
+
+For development purposes we are going to be targeting `iam.dev.canonical.com` and use it as our OIDC provider
+
+the credentials are available in the team LastPass account and need to be swapped in the [configmap.yaml](https://github.com/canonical/identity-platform-admin-ui/blob/main/deployments/kubectl/configMap.yaml#L29-L30) file
```

Comment:
Can we populate the configmap from env vars (https://skaffold.dev/docs/environment/templating/)? Having to edit a committed file can be a little annoying.

Not sure if this will work

Reply:
@nsklikas spent a bit of time on this, so long story short there is not a quick way

to get it working we should mvoe our manifests to use helm, create some values file and then we should be able to specify those via skaffold which would source them from the env

reason is only some of the skaffold fields are templatable, and those are mainly helm related see https://skaffold.dev/docs/environment/templating/

`skaffold.env` doesn't do the trick, not even via kustomize which is what i tried before, i ll poke around another bit and let you know but as it stays not sure i have a quick fix